app/src/main/java/com/duckduckgo/app/statistics/api/OfflinePixelSender.kt

```diff
@@ -87,4 +93,44 @@ class OfflinePixelSender @Inject constructor(
             }
         }
     }
+
+    private fun sendUncaughtExceptionsPixel(): Completable {
+        return defer {
+
+            val pixels = mutableListOf<Completable>()
+            val exceptions = runBlocking { uncaughtExceptionRepository.getExceptions() }
```

Comment:
I don't enjoy the use of `runBlocking` in this function, but I think it might be necessary (and acceptable) since it's bridging `coroutines` and `RxJava` worlds. It's ok to block since we're already running in an `RxJava` IO scheduler